classes/02class/exercises/c02-network02/raghunadhpokkalath/README.md

```diff
@@ -0,0 +1,25 @@
+# c02-network02
+
+## Commands Execution Output
+
+- Answers for `10.0.0.0/16`:
+
+|Subnet|Network|Host Range|Usable Hosts|Broadcast|AWS Reserved|
+|-|-|-|-|-|-|
+|subnet-a|10.0.0.0/18 | 16379 |10.0.0.4 - 10.0.63.254 | 10.0.63.255|10.0.0.1,10.0.0.2,10.0.0.3 
+|subnet-b|10.0.64.0/18|16379|10.0.64.4 - 10.0.127.254 |10.0.127.255 |10.0.64.1,10.0.64.2,10.0.64.3 
+|subnet-c|10.0.128.0/18|16379 |10.0.128.4 - 10.0.191.254 | 10.0.191.255 |10.0.64.1,10.0.64.2,10.0.64.3
```

Comment:
the aws reserved IPs for subnet b and c are the same. Could you fix the subnet c ones?

Reply:
sorry typo i have fixed it